src/test/java/com/github/tomakehurst/wiremock/RemoveStubMappingsAcceptanceTest.java

```diff
@@ -0,0 +1,233 @@
+/*
+ * Copyright (C) 2016-2025 Thomas Akehurst
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.github.tomakehurst.wiremock;
+
+import static com.github.tomakehurst.wiremock.client.WireMock.aResponse;
+import static com.github.tomakehurst.wiremock.client.WireMock.get;
+import static com.github.tomakehurst.wiremock.client.WireMock.listAllStubMappings;
+import static com.github.tomakehurst.wiremock.client.WireMock.removeStubs;
+import static com.github.tomakehurst.wiremock.client.WireMock.saveAllMappings;
+import static com.github.tomakehurst.wiremock.client.WireMock.stubFor;
+import static com.github.tomakehurst.wiremock.client.WireMock.urlEqualTo;
+import static org.hamcrest.MatcherAssert.assertThat;
+import static org.hamcrest.Matchers.containsInAnyOrder;
+import static org.hamcrest.Matchers.is;
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+import com.github.tomakehurst.wiremock.core.WireMockApp;
+import com.github.tomakehurst.wiremock.stubbing.StubMapping;
+import java.io.File;
+import java.util.Arrays;
+import java.util.List;
+import java.util.Objects;
+import java.util.UUID;
+import org.junit.jupiter.api.AfterEach;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+class RemoveStubMappingsAcceptanceTest extends AcceptanceTestBase {
+
+  File rootDir;
+
+  @BeforeEach
+  public void init() throws InterruptedException {
+    rootDir = setupServerWithTempFileRoot();
+    super.init();
```

Comment:
Are we confident this isn't already done by the test framework due to the `@BeforeEach` annotation on the super method?

Reply:
yeah, the temp file created by that method is not available to tests in subclasses. also, AcceptanceTestBase sets up the server in the `@BeforeAll` method, not the `@BeforeEach` method (which only resets the wm server)

Reply:
ah, i see. it's an explicit override of the super method, so would the overridden code even be available to junit to run?

Reply:
ive renamed the method so should remove any ambiguity.

Reply:
Hmmm...that seems to have broken the tests unfortunately